README.md

```diff
@@ -361,7 +361,15 @@ to set `true`; it's effectively a shortcut for `foo=true`).
 - `loops` -- optimizations for `do`, `while` and `for` loops when we can
   statically determine the condition
 
-- `unused` -- drop unreferenced functions and variables
+- `unused` -- drop unreferenced functions and variables (simple direct variable
+  assignments do not count as references unless set to `"keep_assign"`)
+
+- `toplevel` -- drop unreferenced functions (`"funcs"`) and/or variables (`"vars"`)
+  in the toplevel scope (disabled by default)
+
+- `top_retain` -- an array of names to prevent specific toplevel functions
+  and variables from removal even if unreferenced (has no effect when `toplevel`
+  is `false`)
```

Comment:
I've had no end of trouble with the pure_funcs CLI syntax: https://github.com/mishoo/UglifyJS2/issues/1265

I filed and closed two tickets on the thing because I keep forgetting the syntax.